lib/facade_converter.ts

```diff
@@ -85,6 +85,31 @@ export class FacadeConverter extends base.TranspilerBase {
     return handler && !handler(pa);
   }
 
+  visitTypeName(typeName: ts.EntityName) {
+    if (typeName.kind !== ts.SyntaxKind.Identifier) {
+      this.visit(typeName);
+      return;
+    }
+    var ident = base.ident(typeName);
+    if (this.tc) {
+      var symbol = this.tc.getSymbolAtLocation(typeName);
+      if (!symbol) {
+        this.reportMissingType(typeName, ident);
+        return;
+      }
+
+      if (this.getFileAndName(symbol) !== null) {
```

Comment:
extract the value to a var, so you don't have to getFileAndName twice? I expect it to be not entirely cheap.